### PR TITLE
fix colormaps import for older mpl

### PR DESCRIPTION
**Pull request by md-arif-shaikh**

### CODE REVIEW

File: gw_eccentricity/plot_settings.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
 """Fancy settings for plots."""
 from matplotlib import rc
 from cycler import cycler
-from matplotlib import colormaps
 
-dark2 = colormaps["Dark2"].colors
+# Getting colormaps in matplotlib has changed from v3.5.0
+import matplotlib
+if matplotlib.__version__ < "3.5.0":
```
Comment (vijayvarma392) [inline gw_eccentricity/plot_settings.py:7]: Does this always work?
Is something in this discussion relevant for this: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python?

Comment (md-arif-shaikh) [reply gw_eccentricity/plot_settings.py:7]: Seems like using `packaging` would be better. We need this dependency in our package in that case. Is that okay?

Comment (vijayvarma392) [reply gw_eccentricity/plot_settings.py:7]: Yeah, unclear if there is clear winning strategy :).
I suggest maybe some googling to see what people think, but whichever way you think is best is fine with me.

Comment (md-arif-shaikh) [reply gw_eccentricity/plot_settings.py]: I have used `packaging` to compare the versions

**Review by vijayvarma392 — APPROVED** [review]

**Comment by vijayvarma392** [comment]
Please make sure to close the issue with a comment there.

**Pull request merged**